src/haven/instrument/robot.py

```diff
@@ -106,9 +106,9 @@ def load_robot_coros(config=None):
     # Load PV's from config
     if config is None:
         config = load_config()
-    robots = config["robot"]
-    for name, cfg in robots.items():
-        yield make_device(Robot, name=name, labels={"robots"}, prefix=cfg["prefix"])
+        robots = config.get("robot")
+        for name, cfg in robots.items():
```

Comment:
These last two lines should not be indented. They should run whether ``config is None`` is true or false.